tests/test_mesos_tools.py

```diff
@@ -685,20 +668,18 @@ def test_get_tasks_from_app_id():
         ret = mesos_tools.get_tasks_from_app_id('app_id')
         mock_get_running_tasks_from_frameworks.assert_called_with('app_id')
         expected = [mock_task_1, mock_task_2, mock_task_3]
-        assert len(expected) == len(ret) and sorted(ret) == sorted(expected)
+        assert len(expected) == len(ret) and _ids(ret) == _ids(expected)
```

Comment:
What's the reason for this? Is the behaviour of `sorted` changed or have I just been doing this wrong?